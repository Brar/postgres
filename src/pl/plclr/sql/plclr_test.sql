-- first test some basic functionality
CREATE EXTENSION IF NOT EXISTS plclr;

-- simple function to get the module loaded
CREATE FUNCTION plclr_regress_simple() RETURNS int AS '/* ðŸ˜‡ */return 1;' LANGUAGE plclr;

select plclr_regress_simple();