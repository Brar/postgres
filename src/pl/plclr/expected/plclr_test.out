-- first test some basic functionality
CREATE EXTENSION IF NOT EXISTS plclr;
NOTICE:  extension "plclr" already exists, skipping
-- simple function to get the module loaded
CREATE FUNCTION plclr_regress_simple() RETURNS int AS '/* ðŸ˜‡ */return 1;' LANGUAGE plclr;
select plclr_regress_simple();
 plclr_regress_simple 
----------------------
                    1
(1 row)

